hikari/impl/entity_factory.py

```diff
@@ -1055,6 +1055,27 @@ def deserialize_member(
             is_pending=payload["pending"] if "pending" in payload else undefined.UNDEFINED,
         )
 
+    def deserialize_membership_gate(self, payload: data_binding.JSONObject) -> guild_models.MembershipGate:
+        fields: typing.List[guild_models.MembershipGateField] = []
+        for field_payload in payload["form_fields"]:
+            fields.append(
+                guild_models.MembershipGateField(
+                    type=guild_models.VerificationGateFieldType(field_payload["field_type"]),
+                    label=field_payload["label"],
+                    values=field_payload["values"] if "values" in field_payload else [],
+                    is_required=field_payload["required"],
+                )
+            )
```

Comment:
shouldn't this be a list comprehension?

Reply:
This is a bit of an unspoken style thing we've been semi-following here but we've been avoiding list comprehensions that cover more than a couple of lines at mostly cause that can lead to messy looking code